trailblazer/store/crud/update.py

```diff
@@ -138,7 +138,7 @@ def _update_analysis_from_tower_output(
         analysis.logged_at = datetime.now()
         self.delete_analysis_jobs(analysis)
         jobs: list[dict] = tower_api.get_jobs(analysis.id)
-        self.update_analysis_jobs(analysis=analysis, jobs=jobs)
+        self.update_analysis_jobs(jobs)
```

Comment:
I think we probably need to keep the analysis as a parameter here or at least in some way be able to access its analysis.jobs property.